benchmark/utils/main.swift

```diff
@@ -130,6 +130,16 @@ private func registerBenchmark(_ bench: BenchmarkInfo) {
   registeredBenchmarks.append(bench)
 }
 
+@inline(__always)
+private func registerBenchmark(
+  _ name: String,
+  _ function: @escaping (Int) -> (),
+  _ tags: [BenchmarkCategory]
+) {
+  registerBenchmark(
+    BenchmarkInfo(name: name, runFunction: function, tags: tags))
```

Comment:
@atrick 's point with BenchmarkInfo is to slowly deprecate the 'main' file / migrate towards having the tags inside the benchmark's source file. replacing addTo with this runs counter to that.

Reply:
I think this change is fine. I just need to provide new instructions to the one person who was using --registered.